### PR TITLE
User Offboarding Document Updates

### PR DESCRIPTION
**Pull request by zeroxiii**
Closes #

## Summary

**[Remove User Documentation](https://docs.pantheon.io/guides/account-mgmt/workspace-sites-teams/teams#remove-a-user)** - Merged User Offboarding documentation back into main documentation since feature is GA

## Effect

The following changes are already committed:

* Removed EA documentation
* Merged relevant documentation into main workspace and team documentation
* Added additional images

## Remaining Work and Prerequisites

N/A

### Dependencies and Timing

N/A

**Release**:
- [x] When ready
--------------------------------------------------

## Post Launch

- [ ] Redirect `/workspace-offboarding` => `/guides/account-mgmt/workspace-sites-teams/teams` 
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)

### CODE REVIEW

File: source/content/guides/account-mgmt/workspace-sites-teams/07-teams.md [hunk 1]
```diff
@@ -262,6 +280,24 @@ Workspace Administrators, Users in Charge, or Site Owners can add a [Supporting
...
 
+## Remove a Supporting Workspace from a Site
+
+1. [Go to the Site Dashboard](/guides/account-mgmt/workspace-sites-teams/sites#site-dashboard) with the Supporting Workspace you wish to remove.
+
+1. Click **<span class="glyphicons glyphicons-group"></span> Team**.
+
+1. Click the **x** to the right of the Supporting Workspace you wish to remove.
+
+1. If you are a Diamond or Platinum contract customer, you will be provided additional options to remove the user's access from associated sites and will need to select one of the following:
```
Comment (stevector) [inline source/content/guides/account-mgmt/workspace-sites-teams/07-teams.md:291]: @rachelwhitton I saw you just push the word `contract`. But my understanding was that this functionality is available to customers on any contract. Not just Diamond or Platinum contracts.

**Review by stevector — CHANGES_REQUESTED** [review]

File: source/content/guides/account-mgmt/workspace-sites-teams/07-teams.md (hunk 1 above)
Comment (stevector) [reply source/content/guides/account-mgmt/workspace-sites-teams/07-teams.md:291]: cc @zeroxiii

Comment (rachelwhitton) [reply source/content/guides/account-mgmt/workspace-sites-teams/07-teams.md:291]: Slack discussion internally indicates it's available to contract workspaces with Diamond and Platinum support tiers only

Comment (rachelwhitton) [reply source/content/guides/account-mgmt/workspace-sites-teams/07-teams.md]: My mistake, fixed in most recent commit

**Review by rachelwhitton — APPROVED** [review]

**Review by stevector — APPROVED** [review]

**Pull request merged**